src/Tile.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+const colour: string[] = ['none', 'lighBlue', 'yellow', 'orange', 'darkBlue', 'green', 'red', 'purple']
```

Comment:
Why do we define color here aswell as above?

Reply:
cuz we not racist